### PR TITLE
Upgrades to Node18

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx [hunk 1]
```diff
@@ -55,6 +52,14 @@ const ExpandableCard = ({
...
   const transitionRef = useRef<HTMLElement | null>(null);
 
+  const previousDefaultOpen = usePrevious(defaultOpen);
+
+  useEffect(() => {
+    if (previousDefaultOpen !== defaultOpen) {
+      setIsOpen(defaultOpen);
+    }
+  }, [defaultOpen, previousDefaultOpen]);
+
```
Comment (TheSonOfThomp) [inline packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:62]: @bruugey Can we move this to a separate PR?

Comment (bruugey) [reply packages/expandable-card/src/ExpandableCard/ExpandableCard.tsx:62]: yeah i think i must have just caught it here

**Review by TheSonOfThomp — APPROVED** [review]
